feature/platform/fabric/otg_tests/fabric_redundancy_test/README.md

```diff
@@ -0,0 +1,72 @@
+# gNMI-1.14 Fabric redundnacy test
+
+## Summary
+- collect inventory data for each fabric card
+- Verify last restart time is updated
+- verify trassic could be forwarded with only one Fabric Card active.
+
+## Procedure
+### topology and basic setup
+*  Connect OTG port1 to DUT port1 and OTG port2 to DUT port2
+*  Configure IPv6 addresses on both links
+### test 1 Fabric inventory
+
+> NOTE: Theis test is practically identical to Fabric part of
+> https://github.com/openconfig/featureprofiles/tree/main/feature/platform/tests/telemetry_inventory_test;
+> Hence code could be reused and then removed form telemetry_inventory_test.
```

Comment:
let's maybe not overcomplicate? Let's remove this note at all and just either keep this in the telemetry test or add here, or keep in both.

Reply:
Whoever will implement may remove it. After considering if he want take it out of telemetry test.

The telemetry test is one big atomic PASS/FAIL. It do not line-up with WBB Catalog features.
Also we do not know, maybe we will get in future device w/o fabric at all but with 2 x controller-card. or vice-a-versa?

Reply:
We define the rules and need specify exactly what to be done. Less ambiguity for them, better results for us.